src/test/java/com/esri/core/geometry/TestEstimateMemorySize.java

```diff
@@ -0,0 +1,108 @@
+package com.esri.core.geometry;
+
+import com.esri.core.geometry.ogc.OGCConcreteGeometryCollection;
+import com.esri.core.geometry.ogc.OGCGeometry;
+import com.esri.core.geometry.ogc.OGCLineString;
+import com.esri.core.geometry.ogc.OGCMultiLineString;
+import com.esri.core.geometry.ogc.OGCMultiPoint;
+import com.esri.core.geometry.ogc.OGCMultiPolygon;
+import com.esri.core.geometry.ogc.OGCPoint;
+import com.esri.core.geometry.ogc.OGCPolygon;
+import org.junit.Test;
+import org.openjdk.jol.info.ClassLayout;
```

Comment:
@mbasmanova Could you add a comment here for now for the reference
//ClassLayout is GPL with Classpath exception, see http://openjdk.java.net/legal/gplv2+ce.html

Reply:
@stolstov Sure. Added the suggested comment.